shared/actions/chat.js

```diff
@@ -2054,7 +2059,10 @@ function * _getInboxAndUnbox ({payload: {conversationIDKey}}: Constants.GetInbox
   const {conversations} = result
   if (conversations && conversations[0]) {
```

Comment:
So this call really should flow exactly like the full loadInbox but i didn't want to refactor that much code. the long story short is this individual call can be like the full inbox load for a single item, where we can get a success or failure. The failure can include a rekeyInfo that's new which we should be updating. This may not happen in practice but maybe its something we'd clean up later.